### ISSUE TITLE
ipadnsrecord does not create A record and PTR with record_value and create_reverse

### ISSUE THREAD

**Issue by jonaz** [freeipa/ansible-freeipa#491]
```
- name: Add configured DNS records in IPA DNS
  freeipa.ansible_freeipa.ipadnsrecord:
    ipaadmin_password: "{{ipaadmin_password}}"
    name: "test"
    zone_name: "zone1.domain.com."
    record_type: "A"
    record_value: "10.10.10.10"
    create_reverse: yes
  become: True
```

This results in only PTR record created and no A record to `test.zone1.domain.com`

If i use the folloing instead both PTR and A records are created successfully. Should the above not work?

```
- name: Add configured DNS records in IPA DNS
  freeipa.ansible_freeipa.ipadnsrecord:
    ipaadmin_password: "{{ipaadmin_password}}"
    name: "test"
    zone_name: "zone1.domain.com."
    a_ip_address: "10.10.10.10"
    a_create_reverse: yes
  become: True
```

**Comment by rjeffman** [comment]
Hello @jonaz, this behavior is expected.

The first playbook uses the compatibility mode to Ansible's Community General ipa_dnsrecord plugin, which does not support `create_reverse`. The second playbook uses ansible-freeipa's method to define the record, and `create_reverse` is supported only in this method.

**Comment by rjeffman** [comment]
@jonaz thank you for the report.  There is a PR (#493) to fix this issue.

**Comment by rjeffman** [comment]
A fix for this issue has been merged as is available in 0.3.2 release.
